### ISSUE TITLE
Commands to add

### ISSUE THREAD

**Issue by Nodd** [Nodd/spyderplugins.vim#1]
List from https://github.com/spyder-ide/spyder/issues/669#issuecomment-222555709:

- [ ] %
- [ ] {
- [ ] }
- [ ] gg
- [ ] GG
- [ ] $
- [ ] 0
- [ ] :$
- [ ] :^
- [ ] :r [!]
- [ ] yy
- [ ] p
- [ ] m
- [ ] "
- [ ] Shift+V
- [ ] Ctrl+V
- [ ] A (CAPITAl) and I (capital) in *+V modes
- [ ] :s
- [ ] &
- [ ] q / "macro" bindings
- [ ] gU
- [ ] gu
- [ ] ga
- [ ] [number] dd
- [ ] f
- [ ] F
- [ ] *
- [ ] /
- [ ] :hls
- [ ]  "folding" bindings (Note: folding is not possible in spyder right now)

**Comment by Nodd** [comment]
I added a few more. This is going to be a looong process...
And I'm sure I didn't get all the details and corner cases right (exact cursor position after the action, word definition, before or after whitespace...)

**Comment by mannyglover** [comment]
- [x] w [go to next word]
- [ ] b [go to beginning of previous word]
- [ ] e [go to end of current word (or next word if cursor is currently on whitespace)]
- [ ] B [go to beginning of previous "word," where word is defined as, I believe, any string of consecutive non-whitespace characters.]
- [ ] E [E:e::B:b <- standardized test analogy explanation]

**Comment by sacherus** [comment]
d$ -> delete till end
^ -> go to the beginning of the blank line

**Comment by mvds314** [comment]
If I can make the time, I can try to implement a few of those commands. Not sure who's in charge of this project. Can I just make a PR?

**Comment by ccordoba12** [comment]
@mvds314, yes, please do! @ok97465 helps us to maintain it but we can also give review any PR you want to contribute.

**Comment by ok97465** [comment]
> If I can make the time, I can try to implement a few of those commands. Not sure who's in charge of this project. Can I just make a PR?

Sorry for the late response. Notification for this repository was turned off. 
Please submit a PR at your convenience.

**Comment by luis-j** [comment]
One of the first (if not *the* first) command I would wish is the dot command to repeat the last action. I'm afraid my knowledge of Python is extremely limited, and I wouldn't be able to contribute code.

**Comment by Masterhigure** [comment]
Several replies here with sensible suggestions. Maybe it would be sensible to edit the original / top comment to contain those as well? So that all the wanted commands are gathered in one post rather than spread out over a thread?
